doc/changelog.md

```diff
@@ -1 +1 @@
-../CHANGELOG.md
```

Comment:
I'm not sure what happened here, but this new commit breaks the symlink from `doc/changelog.md` to `CHANGELOG.md`. You should be able to change this on your branch by doing:
```
git checkout HEAD^ doc/changelog.md
git add doc/changelog.md
git commit --amend
```
Everything else is great! Just this one fix and we'll merge :)